ui/app/AppLayouts/Wallet/services/dapps/DAppsRequestHandler.qml

```diff
@@ -34,7 +34,7 @@ QObject {
     signal displayToastMessage(string message, bool error)
     signal sessionRequestResult(/*model entry of SessionRequestResolved*/ var request, bool isSuccess)
     signal maxFeesUpdated(real maxFees, int maxFeesWei, bool haveEnoughFunds, string symbol)
-    signal estimatedTimeUpdated(int minMinutes, int maxMinutes)
```

Comment:
This was actually better; why not continue using it?

Reply:
To avoid returning complex data from nim to QML. I will do it though :))